babel.config.json

```diff
@@ -0,0 +1,115 @@
+{
+  "babelrcRoots": [
+    ".",
+    "./packages/*"
+  ],
+  "env": {
+    "production": {
+      "presets": [
+        "@babel/preset-env",
+        "@babel/preset-react"
+      ],
+      "plugins": [
+        "@babel/plugin-syntax-dynamic-import",
+        "@babel/plugin-syntax-import-meta",
+        "@babel/plugin-proposal-class-properties",
+        "@babel/plugin-proposal-json-strings",
+        [
+          "@babel/plugin-proposal-decorators",
+          {
+            "legacy": true
+          }
+        ],
+        "@babel/plugin-proposal-function-sent",
+        "@babel/plugin-proposal-export-namespace-from",
+        "@babel/plugin-proposal-numeric-separator",
+        "@babel/plugin-proposal-throw-expressions",
+        "@babel/plugin-proposal-export-default-from",
+        "@babel/plugin-proposal-logical-assignment-operators",
+        "@babel/plugin-proposal-optional-chaining",
+        [
```

Comment:
Agreed, we could add only the features we currently use

Reply:
These will probably be mostly redundant when using typescript.
babel deprecated the yearly presents, and these plugins were added using the `babel-upgrade` script.  These are the latest versions of the plugins that we already had.

Reply:
If we look at baseweb, looks like there's minimal configuration by using the react preset. Perhaps we can try what's there first?

Reply:
My approach for tasks like this is to migrate first, and optimize later as it keeps diffs smaller and it more apparent on what the optimizations are.

Either way, this won't work until we fix the tests since the tests are referencing 'showcase' and it isn't being "compiled"

Reply:
trimmed to
```
 "presets": [
        "@babel/preset-env",
        "@babel/preset-react"
      ],
      "plugins": [
        "@babel/plugin-syntax-dynamic-import",
        "@babel/plugin-proposal-class-properties",
        "@babel/plugin-proposal-export-default-from",
        "@babel/plugin-proposal-optional-chaining",
        "@babel/plugin-proposal-nullish-coalescing-operator"
      ]
```

Reply:
actually just removed `dynamic-import` as well.